astropy/coordinates/builtin_frames/gcrs.py

```diff
@@ -70,8 +71,8 @@ class GCRS(BaseCoordinateFrame):
     default_representation = SphericalRepresentation
 
     obstime = TimeFrameAttribute(default=DEFAULT_OBSTIME)
-    obsgeoloc = QuantityFrameAttribute(default=0, unit=u.m, shape=(3,))
-    obsgeovel = QuantityFrameAttribute(default=0, unit=u.m/u.s, shape=(3,))
+    obsgeoloc = CartesianRepresentationFrameAttribute(default=0, unit=u.m)
```

Comment:
Per previous comment, this could be `CartesianRepresentationFrameAttribute(default=[0, 0, 0], unit=u.m)` (I think).